server/clearbit.js

```diff
@@ -479,7 +479,7 @@ export default class Clearbit {
     const hullUser = this.hull.asUser({ email: person.email, anonymous_id: `clearbit-prospect:${person.id}` });
     const domain = company_traits["clearbit_company/domain"];
 
-    hullUser.logger.info("incoming.user.success", { person, source: "prospector" });
+    hullUser.logger.info("incoming.user.success", { source: "prospector" });
```

Comment:
Same here, please add the data in the log so we can debug what information came from the prospector.